test/suite-functional/api/functional.thenable.spec.php

```diff
@@ -36,10 +36,10 @@
         }
     });
 
-    rit('that does reject with non throwable', function () {
+    rit('resumes the strand with a value when rejected', function () {
```

Comment:
This resumes the strand with an exception when the promise is rejected with a non-exception.